query/select.go

```diff
@@ -288,6 +288,28 @@ func (b *exprIteratorBuilder) buildCallIterator(ctx context.Context, expr *influ
 				}
 			}
 			return newMovingAverageIterator(input, int(n.Val), opt)
+		case "exponential_moving_average":
+			n := expr.Args[1].(*influxql.IntegerLiteral)
+			if n.Val > 1 && !opt.Interval.IsZero() {
+				if opt.Ascending {
+					opt.StartTime -= int64(opt.Interval.Duration) * (n.Val - 1)
+				} else {
+					opt.EndTime += int64(opt.Interval.Duration) * (n.Val - 1)
+				}
+			}
+
+			nHold := -1
+			if len(expr.Args) >= 3 {
+				nHold = int(expr.Args[2].(*influxql.IntegerLiteral).Val)
+			}
+
+			var warmupType string
+			if len(expr.Args) >= 4 {
+				warmupType = expr.Args[3].(*influxql.StringLiteral).String()
```

Comment:
There's no need to do this. Just do `expr.Args[3].(*influxql.StringLiteral).Val` and you won't have to strip the quotes.